File: CharChem/Python/CharChem/math/Rect.py
class Rect:
	""" Rectangle object """
	__slots__ = ('A', 'B')
	def __init__(self):
		self.A = None
		self.B = None
	def init(self, xa, ya, xb, yb):
		""" Init rect by number coordinates """
		self.A.init(xa, ya)
		self.B.init(xb, yb)

File: CharChem/TypeScript/CharChem/math/Rect.ts
// Rectangle object
export class Rect {
	public A: Point;
	public B: Point;
	public init(xa: number, ya: number, xb: number, yb: number) {
		this.A.init(xa, ya);
		this.B.init(xb, yb);
	}
}

